backbone/src/main/java/org/researchstack/backbone/task/OrderedTaskFactory.java

```diff
@@ -0,0 +1,643 @@
+package org.researchstack.backbone.task;
+
+import android.content.Context;
+
+import org.researchstack.backbone.R;
+import org.researchstack.backbone.answerformat.AnswerFormat;
+import org.researchstack.backbone.answerformat.ChoiceAnswerFormat;
+import org.researchstack.backbone.model.Choice;
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.step.CompletionStep;
+import org.researchstack.backbone.step.NavigationCustomQuestionStep;
+import org.researchstack.backbone.step.active.AccelerometerRecorderConfig;
+import org.researchstack.backbone.step.active.ActiveStep;
+import org.researchstack.backbone.step.active.CountdownStep;
+import org.researchstack.backbone.step.InstructionStep;
+import org.researchstack.backbone.step.Step;
+import org.researchstack.backbone.step.active.DeviceMotionRecorderConfig;
+import org.researchstack.backbone.step.active.NavigationActiveStep;
+import org.researchstack.backbone.utils.ResUtils;
+import org.researchstack.backbone.utils.StepResultHelper;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Created by TheMDP on 2/4/17.
+ *
+ * In iOS, they included a bunch of static methods for building OrderedTasks in the
+ * OrderedTask class.  However, I think they belong in this Factory class
+ */
+
+public class OrderedTaskFactory {
```

Comment:
Not sure why this class wasn't showing up in my diff online, commenting here so you guys see it.